src/main/java/seedu/module/logic/commands/TagCommand.java

```diff
@@ -35,17 +36,15 @@
     public static final String MESSAGE_DUPLICATE_TASK = "This tag already exists.";
 
     private final Index index;
-    private final Tag tag;
+    private Set<Tag> tags;
 
     /**
      * @param index of the person in the filtered person list to edit the remark
-     * @param tag of the person to be added
      */
-    public TagCommand(Index index, Tag tag) {
-        requireAllNonNull(index, tag);
+    public TagCommand(Index index) {
```

Comment:
Personally, I prefer to initialise both index and tags in the constructor. But I'll leave it up to you to decide.

Reply:
This is what I thought and planned to do, but I implement this way is to achieve consistency with editCommand.

Reply:
Documentation (UG) updated